src/elements/content-sidebar/activity-feed/version/Version.js

```diff
@@ -38,17 +41,25 @@ const ACTION_MAP = {
 const Version = (props: Props): React.Node => {
     // $FlowFixMe
     const action = selectors.getVersionAction(props);
-    const { id, intl, onInfo, version_number, version_promoted } = props;
+    const { id, intl, onInfo, version_number, version_promoted, uploader_display_name } = props;
     // $FlowFixMe
-    const { name } = selectors.getVersionUser(props);
-
+    const user = selectors.getVersionUser(props);
+    let display_name = user.name;
```

Comment:
We prefer to use destructuring to rename properties rather than rename-only reassignments. We should also use camelCase for all variable names.